docs/content/en/docs/install/install.md

```diff
@@ -76,67 +88,104 @@ Some helpful hints:
+(documented in its own README file),
+that defines configurations specific to that component.
+
+All configuration changes for all components
+are made to the `keptn/values.yaml` file.
+This is discussed more in
+[Modify Helm configuration options](#modify-helm-configuration-options)
+below.
+
+The following table summarizes the Keptn `values.yaml` files.
+
+* Click the name in the "Component" column
+  to view the README file that documents the Helm values
+  that can be set for that component
+* Click the string in the "Configuration file" column
+  to view the actual `values.yaml` file for each
+
+| Component                                                                                                                  | Used for                                                                                                                  | Configuration file |
+|----------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------| --------------------|
+| [Keptn](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn/README.md)                           | Installs subcharts, global configuration                                                                                  | [keptn/values.yaml](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn/values.yaml) |
+| [lifecycle-operator](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-lifecycle-operator/README.md) | [Observability](../implementing/otel.md), [Release Lifecycle Management](../intro/_index.md#release-lifecycle-management) | [keptn-lifecycle-operator/values.yaml](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-lifecycle-operator/values.yaml) |
+| [metrics-operator](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-metrics-operator/README.md)    | [Keptn metrics](../implementing/evaluatemetrics.md), [Analysis](../implementing/slo.md)                                   | [keptn-metrics-operator/values.yaml](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-metrics-operator/values.yaml) |
+| [cert-manager](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-cert-manager/README.md)            | [TLS Certificate management for all Keptn components](../architecture/cert-manager)                                                                       | [keptn-cert-manager/values.yaml](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-cert-manager/values.yaml) |
+
 ## Customizing the configuration of components
 
 To modify configuration of the subcomponents of Keptn,
-use the corresponding Helm values in your main `values.yaml`.
+use the corresponding Helm values in your `keptn/values.yaml` file.
```

Comment:
this always looks like a path. but we don't have that path anywhere really. i suggest not to use it
```suggestion
use the corresponding Helm values in your `values.yaml` file.
```